controllers/slicegateway/slicegateway.go

```diff
@@ -678,3 +680,69 @@ func (r *SliceGwReconciler) SyncNetOpConnectionContextAndQos(ctx context.Context
 	debugLog.Info("netop pods sync complete", "pods", r.NetOpPods)
 	return nil
 }
+
+func (r *SliceGwReconciler) createHeadlessServiceForGwServer(slicegateway *kubeslicev1beta1.SliceGateway) *corev1.Service {
+	svc := &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      slicegateway.Status.Config.SliceGatewayRemoteGatewayID,
```

Comment:
probably a good idea to prefix the service name with slice name. Otherwise when you run `k get svc` it is difficult to determine which service belongs to which slice

Reply:
yeah, makes sense!